disable gperf tests to avoid cross compiled binaries running on host

--- ACE_wrappers_org/apps/gperf/Makefile.am	2015-02-06 15:03:45.543287039 +0530
+++ ACE_wrappers/apps/gperf/Makefile.am	2015-02-06 15:04:28.043240180 +0530
@@ -11,8 +11,7 @@
 
 SUBDIRS = \
         . \
-        src \
-        tests
+        src 
 
 ## Makefile.gperf_docs.am
 
